src/components/FieldWrapper/FieldWrapper.js

```diff
@@ -49,11 +49,14 @@ FieldWrapper.propTypes = {
     onClear: PropTypes.func,
     /** to style the field as it is focused */
     isFocused: PropTypes.bool,
+    /** defines if the component is disabled */
```

Comment:
Perhaps it is a good opportunity to move this component to TS if you are already changing it.